### PR TITLE
Added sorting on view full list

### PR DESCRIPTION
**Pull request by d34th4ck3r**

### CODE REVIEW

**Comment by d34th4ck3r** [comment]
@adamkarminski Can you have a look? 

Notice that I have created the pull request on JPN-436.

File: extensions/wikia/CommunityPage/models/CommunityPageSpecialInsightsModel.class.php [hunk 1]
```diff
@@ -61,7 +61,7 @@ private function getInsightModule( $type, $sortingType = self::INSIGHTS_MODULE_S
 		$insightPages['title'] = wfMessage( 'communitypage-' . $type. '-title' )->text();
 		$insightPages['description'] =  wfMessage( 'communitypage-' . $type. '-description' )->text();
 
-		$insightPages['fulllistlink'] = SpecialPage::getTitleFor( 'Insights', $type )->getLocalURL();
+		$insightPages['fulllistlink'] = SpecialPage::getTitleFor( 'Insights', $type )->getLocalURL() . "?sort=" . self::INSIGHTS_MODULE_SORT_TYPE;
```
Comment (lukaszkonieczny) [inline extensions/wikia/CommunityPage/models/CommunityPageSpecialInsightsModel.class.php]: You can pass url parameters as an array to `getLocalURL()` method

see: https://github.com/Wikia/app/blob/dev/includes/Title.php#L1442

**Comment by d34th4ck3r** [comment]
@lukaszkonieczny Changed according to function definition. Please check again.

**Comment by lukaszkonieczny** [comment]
LGTM

**Pull request merged**